deployments/apiv2/services/users/src/main.py

```diff
@@ -308,8 +308,7 @@ async def register(
                         # other customer accounts will be leaked
                         failed_msg = "Username already in use"
                     elif "users_email_key" in str(e):
-                        # Don't want to leak emails of users under other customer accounts, so return default msg
```

Comment:
you could probably just remove this block and have the else block that already states "Account registration failed"